### PR TITLE
simplify example to fetch user/org ID from ocm

### PR DESCRIPTION
**Pull request by jbpratt**
simplify the incantation for fetching the user id and org id from ocm token

### CODE REVIEW

**Review by machi1990 — CHANGES_REQUESTED** [review]
added suggestions to add missing closing paretheses

**Comment by jbpratt** [comment]
thanks for catching that @machi1990 :)

File: kas-installer-defaults.env [hunk 1]
```diff
@@ -6,12 +6,12 @@ trap 'echo "** Unbound variable must be configured in kas-installer.env or user
 RH_USERNAME="${RH_USERNAME}"
 
 # [required]
-# Owning users RH account_id. See `str=$(ocm token | awk -F'.' '{print $2}')===; echo ${str:0:${#str}&-4} | base64 -d | jq -r .account_id`
+# Owning users RH account_id. See `str=$(ocm token --payload | jq -r .account_id)`
```
Comment (MikeEdgar) [inline kas-installer-defaults.env:9]: I think you can even go a step further and remove the `str=$(` and `)`. The `str` variable was only present to support the bash substring that isn't present anymore.

**Review by MikeEdgar — APPROVED** [review]
lgtm :+1:

**Pull request merged**